src/DebugBar/Storage/RedisStorage.php

```diff
@@ -15,15 +15,15 @@
  */
 class RedisStorage implements StorageInterface
 {
+    /** @var \Predis\Client|\Redis */
     protected $redis;
 
-    protected $hash;
+    protected string $hash;
```

Comment:
major breaking change (non-private property of non-final class). please revert.

Reply:
Fixed